test/commands/run-worker.test.js

```diff
@@ -186,51 +185,4 @@ describe("run-worker command", function() {
                 assertMissingOrEmptyDirectory("build", "run-worker");
             });
     });
-
-    describe("standalone mode", function() {
```

Comment:
so it doesn't work in standalone mode anymore?

Reply:
I'm not entirely familiar with the plugin and am not sure how what standalone mode means or how to test it

This test was removed, because the only assertion it makes is that `app:deploy` is called as a child process to build actions, which the plugin doesn't do anymore since it uses the Runtime lib directly